openpype/hosts/maya/api/plugin.py

```diff
@@ -210,18 +202,14 @@ def update(self, container, representation):
         alembic_data = {}
         if representation["name"] == "abc":
             alembic_nodes = cmds.ls(
-                "{}:*".format(members[0].split(":")[0]), type="AlembicNode"
+                "{}:*".format(namespace), type="AlembicNode"
             )
             if alembic_nodes:
                 for attr in alembic_attrs:
                     node_attr = "{}.{}".format(alembic_nodes[0], attr)
                     alembic_data[attr] = cmds.getAttr(node_attr)
             else:
-                cmds.warning(
-                    "No alembic nodes found in {}".format(
-                        cmds.ls("{}:*".format(members[0].split(":")[0]))
-                    )
-                )
+                self.log.debug("No alembic nodes found in {}".format(members))
```

Comment:
Note that I've changed this to debug logging. I felt the warning was redundant for non-animated alembics (which could be models, or just other static objects) because those will not have an `AlembicNode` attached. @tokejepsen should be fine?